examples/reshare/src/orchestrator/actor.rs

```diff
@@ -346,6 +292,21 @@ where
                     }
                 }
             },
+            result = &mut pending_boundary_fetch => {
```

Comment:
seems we only use the result for logging ... trying to think of a case where listening on this is useful (rather than just enqueueing).